lib/util/pretty-time.ts

```diff
@@ -40,3 +41,44 @@ export function toMs(time: string): number | null {
     return null;
   }
 }
+
+const rangeRegex = regEx(/^(?<operator>(>=|<=|<|>))\s*(?<age>.*)/);
+
+export function satisfiesDateRange(
+  date: string,
+  range: string
+): boolean | null {
+  const grps = range.trim().match(rangeRegex)?.groups;
+  if (!grps) {
+    return null;
+  }
+
+  const { operator, age } = grps;
+  const luxonDate = DateTime.fromISO(date, { zone: 'utc' });
```

Comment:
these iso date strings can have a time shift value, does luxon correctly convert that to UTC? or does it simply ignore the shift value.

if it ignored, then we need to use default parse and convert to UTC.

Reply:
Yes, it converts correctly. I tested it locally.